WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt

```diff
@@ -33,7 +33,7 @@ class ActionableEmptyView : LinearLayout {
     }
```

Comment:
Not really related to this PR but I'm just catching up on all the great work done in this series and thought I would make a comment anyway :)
One of the nice things you can do with Kotlin is overload constructors, so you can simplify this class and get rid of several lines of code by doing: 
```
class ActionableEmptyView @JvmOverloads constructor(
    context: Context, 
    attrs: AttributeSet? = null, 
    defStyleAttr: Int = 0
): LinearLayout(context, attrs, defStyleAttr) {
    init {
        initView(context, attrs)
    }
...
}
```

Reply:
Thanks for the tip, @AmandaRiu!  I used that trick when developing the class originally while I was toying with custom constructors.  I removed those eventually, but never went back to optimize the default ones.  I'll update the `ActionableEmptyView` class in Part 5 of this pull request series.